src/button/internal.tsx

```diff
@@ -134,9 +135,16 @@ export const InternalButton = React.forwardRef(
       [styles['full-width']]: shouldHaveContent && fullWidth,
     });
 
+    const explicitTabIndex =
+      __nativeAttributes && 'tabIndex' in __nativeAttributes ? __nativeAttributes.tabIndex : undefined;
```

Comment:
Could this be simplified to `__nativeAttributes?.tabIndex` ?

Reply:
I can't because of the __nativeAttributes type:

```
(React.HTMLAttributes<HTMLAnchorElement> & React.HTMLAttributes<HTMLButtonElement>) | Record<`data-${string}`, string>
```